### PR TITLE
Allow new-style module layout

### PR DESCRIPTION
**Pull request by athornton**

### CODE REVIEW

File: LSST Catalog Access Tutorial.ipynb [hunk 1]
```diff
@@ -472,9 +477,9 @@
  ],
  "metadata": {
   "kernelspec": {
-   "display_name": "Python 3",
```
Comment (SimonKrughoff) [inline LSST Catalog Access Tutorial.ipynb:475]: I like these using the vanilla python kernel.

Comment (athornton) [reply LSST Catalog Access Tutorial.ipynb:475]: Whoa, really?  If it's an LSST notebook, shouldn't you be using the stack?

I'm planning on dropping the system python from user-visible kernels after the JL2 migration, but.....

Comment (athornton) [reply LSST Catalog Access Tutorial.ipynb:475]: I'll change it back if you prefer, though.

Comment (athornton) [reply LSST Catalog Access Tutorial.ipynb:475]: And, um, all of these except token-info (and Catalog Access) already use LSST:


```
2017_tutorial.ipynb:   "display_name": "LSST",
Firefly.ipynb:   "display_name": "LSST",
LSST Catalog Access Tutorial.ipynb:   "display_name": "LSST",
LSST SODA Tutorial.ipynb:   "display_name": "LSST",
cluster.ipynb:   "display_name": "LSST",
star_galaxy.ipynb:   "display_name": "LSST",
token-info.ipynb:   "display_name": "Python 3",
wise_color_color.ipynb:   "display_name": "LSST",
```

Comment (SimonKrughoff) [reply LSST Catalog Access Tutorial.ipynb:475]: We can have the "What kernel should we use?" conversation later.  I just don't want to change it in an unrelated PR.

**Pull request merged**